active/package-maintainers.md

```diff
@@ -0,0 +1,125 @@
+# Package Maintainers Working Group
+
+## Scope of responsibilities
+
+This group supports package maintainers in the Django ecosystem.
+Packages are an important part of the experience of using Django, and their maintainers can benefit from different forms of support than Django contributors and users.
+
+The goals of the group are:
+
+- To facilitate discussions and share knowledge about common issues or tasks in package maintenance.
+- To coordinate efforts to improve the health and quality of packages in the Django ecosystem.
+- To advocate for the needs of package maintainers in the Django community.
+
+This is a broad definition of the group’s remit.
+Current members select specific activities to undertake based on their interests and availability.
+See [Group activities](#group-activities) for examples of what the group may choose to do.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- No special powers delegated by the board to the group.
+- No specific "Django Software Foundation" actions the group can take directly.
+
+Any request for funding or resources are made via existing processes outside of this group.
+This can be revised in the future as and when the group identifies specific recurring needs.
+
+## Membership
+
+- Chair: You?
+- Co-Chair: You?
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas
+- Other members:
+  - You?
+
+## Eligibility
+
+Membership is open to all Individual Members of the Django Software Foundation.
+
+## How to join
+
+To join, members must express an interest in the #packages channel of the [Django Discord server](https://discord.gg/xcRH6mN4fa), or reach out to a current group member.
```

Comment:
I think a decision should be made if the Discord or the Forum is the "official" place to announce such things. I'm on Discord and sort-of like it, but I think the Forum might be a better place for this, because it's easier to refer back to eventual discussion happening later. It could also give some guidance to interested people since they can see other people's introduction posts.

Reply:
My comment was not about htis working group specifically. I'm just not a big fan of chat-based communication when you should be able to refer to it later. It's really good for chatting, spamming, community building etc. but maybe the working groups would benefit from a more formal space. 

I know the discussions about the forum and how official it is, and I unfortunately don't really have a recommendation.

Reply:
Expressing interest to join isn't necessarily a thing that is  worth archival. Joining on the other hand is probably something of note.